tests/index.js

```diff
@@ -131,3 +131,20 @@ tap.test('should return validation error message if no conversion is possible',
   const response = await fastify.inject().get('/').query({ a: 'hello', b: '2' }).then(res => res.json())
   t.equal(response.message.indexOf('querystring/a'), 0)
 })
+
+tap.test('should fast serialize for the typebox 0.26.0 allOf intersect representation', async t => {
```

Comment:
What's _fast_ serialization?

Reply:
Ah, that's just a indirect `fast-json-stringify` test. The test asserts that the new `allOf` intersect schema representation on TB 0.26.0 doesn't raise `FJS` errors. Previously TypeBox produced an intersected `object` representation which is a bit easier to serialize. Some notes on the representation change can be found at the link below.

https://github.com/sinclairzx81/typebox/blob/master/changelog/0.26.0.md#Intersect-Schema-Representation